pint_pandas/pint_array.py

```diff
@@ -691,7 +723,10 @@ def __array__(self, dtype=None, copy=False):
         return np.array(self._data, dtype=dtype, copy=copy)
 
     def _to_array_of_quantity(self, copy=False):
-        qtys = [self._Q(item, self._dtype.units) for item in self._data]
+        qtys = [
+            self._Q(item, self._dtype.units) if not pd.isnull(item) else item
```

Comment:
You use pd.isna elsewhere to check this condition, and it is supposedly preferred to the alias pd.isnull.  Should you use pd.isna here for consistency?

Reply:
changed it